### DISCUSSION TITLE
When is it available for react native?

### DISCUSSION THREAD

**Discussion by Kstar0722** [rainbow-me/rainbowkit#2131] [Feedback]
Hello, dev team
Thank you for great library.

It was great for react.
I'd like to use this for react native.

Do you have any plan for this?

**Comment by DanielSinclair** [comment]
@Kstar0722 This is currently not planned, but quite a few products have deployed RainbowKit in web browser containers within their React Native apps. Have a look at Warpcast and their Verify flow as an example
